app/components/channel_list/categories/header/index.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+
+import {useTheme} from '@app/context/theme';
+import {typography} from '@app/utils/typography';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    container: {
+        paddingVertical: 8,
+        marginTop: 12,
+    },
+    heading: {
+        color: changeOpacity(theme.sidebarText, 0.64),
+    },
+}));
+
+const textStyle = StyleSheet.create([typography('Heading', 75)]);
```

Comment:
any reason why not to include `...typography('Heading', 75)` in the `heading` style defined above?